subxt/Cargo.toml

```diff
@@ -27,25 +27,28 @@ default = ["jsonrpsee", "native"]
 native = [
     "jsonrpsee?/async-client",
     "jsonrpsee?/client-ws-transport-tls",
+    "jsonrpsee?/ws-client",
     "subxt-lightclient?/native",
     "tokio-util",
-    "reconnecting-jsonrpsee-ws-client?/native",
+    "tokio?/sync",
 ]
 
 # Enable this for web/wasm builds.
 # Exactly 1 of "web" and "native" is expected.
 web = [
     "jsonrpsee?/async-wasm-client",
     "jsonrpsee?/client-web-transport",
+    "jsonrpsee?/wasm-client",
     "getrandom/js",
     "subxt-lightclient?/web",
     "subxt-macro/web",
     "instant/wasm-bindgen",
-    "reconnecting-jsonrpsee-ws-client?/web",
+    "tokio?/sync",
+    "finito?/wasm-bindgen",
 ]
 
 # Enable this to use the reconnecting rpc client
-unstable-reconnecting-rpc-client = ["dep:reconnecting-jsonrpsee-ws-client"]
+unstable-reconnecting-rpc-client = ["dep:finito", "dep:tokio", "jsonrpsee"]
```

Comment:
Does this work in WASM as well?

Reply:
It's already tested in my other library but I can add a test for it in the wasm tests that we already have

Reply:
added a test for it